modules/core-updates/8.0.md

```diff
@@ -212,14 +475,51 @@ Changes related to the [upgrade to Symfony 4.4](https://github.com/PrestaShop/Pr
+
+### Database
+
+* `ps_product.unit_price_ratio` → The reference is now `ps_product.unit_price` ([read more](https://github.com/PrestaShop/PrestaShop/pull/26762)).
+
+## Updated dependencies
+
+### PHP libraries
+
+| Library name                       | Old version                                                   | New version   |
+|------------------------------------|---------------------------------------------------------------|---------------|
+| curl/curl                          | 1.2                                                           | 2.3           |
+| csa/guzzle-bundle                  | 1.3 ([forked](https://github.com/PrestaShop/CsaGuzzleBundle)) | _~ Removed ~_ |
+| cssjanus/cssjanus                  | 1.2 (forked)                                                  | 2.1           |
+| doctrine/cache                     | 1.8                                                           | 2.0           |
+| doctrine/doctrine-bundle           | 1.6                                                           | 2.3           |
+| egulias/email-validator            | 2.1                                                           | _~ Removed ~_ |
+| guzzlehttp/cache-subscriber        | 0.2                                                           | _~ Removed ~_ |
+| guzzlehttp/guzzle                  | 5.3                                                           | 7.3           |
+| mrclay/minify                      | 2.3                                                           | 3.0           |
+| paragonie/random_compat            | 2.0                                                           | _~ Removed ~_ |
+| pelago/emogrifier                  | 2.1                                                           | 5.0           |
+| prestashop/circuit-breaker         | 3.0                                                           | 4.0           |
+| prestashop/translationtools-bundle | 4.0                                                           | 5.0           |
+| shudrum/array-finder               | 1.1                                                           | _~ Removed ~_ |
+| symfony/polyfill-apcu              | 1.0                                                           | _~ Removed ~_ |
+| symfony/polyfill-php73             | 1.10                                                          | _~ Removed ~_ |
+| symfony/symfony                    | 3.4                                                           | 4.4           |
+| twig/twig                          | 1.38                                                          | 3.0           |
+
+### JS libraries
+
+* The file `js/rtl.js` [has been removed](https://github.com/PrestaShop/PrestaShop/pull/26616).
+
+* The following dependencies, not required by the Back office directly, [have been removed](https://github.com/PrestaShop/PrestaShop/pull/25917) from Symfony-based pages:
+  * bootstrap-slider
+  * compass
+  * mobile-detect
+  * sprintf-js
+  * tether
```

Comment:
@PierreRambaud This may interest you, about sprintf-js, as you may want to add it again